src/manager.js

```diff
@@ -574,14 +608,15 @@ AdManager.prototype.asyncRefreshSlot = function (domElement) {
 };
 
 /**
+  * Fetches a new ad for just single dom element
   * Uses the `cmd` async GPT queue to enqueue ad manager function to run
   * if the GPT API is not yet ready.  Assures slots have been configured,
   * etc. prior to trying to make the ad request
   *
   * @param {domElement} DOM element containing the DFP ad slot
   * @returns undefined
 */
-AdManager.prototype.refreshAds = function (domElement) {
+AdManager.prototype.refreshSlot = function (domElement) {
```

Comment:
@spra85 I'm not sure why these two functions were separate, but I think it makes sense to combine them.